ta/arch/arm/link.mk

```diff
@@ -5,6 +5,14 @@ link-script-dep$(sm) = $(link-out-dir$(sm))/.ta.ld.d
 SIGN_ENC ?= $(PYTHON3) $(ta-dev-kit-dir$(sm))/scripts/sign_encrypt.py
 TA_SIGN_KEY ?= $(ta-dev-kit-dir$(sm))/keys/default_ta.pem
 
+# In case the TA offline-signing has been performed.
+ifeq ($(CFG_STITCH_TA),y)
```

Comment:
You shouldn't use a CFG_ prefix due to the recompilation issue it triggers. Same reason we use `EARLY_TA_PATHS` rather than `CFG_EARLY_TA_PATHS` for instance. These variables are intended for the build system and should have no side-effect on the compiled source code.
I realize that `CFG_ENCRYPT_TA` has the same issue :-/

Reply:
Yep, the commit was before you mentioned the purpose of the CFG_XXX variables :)